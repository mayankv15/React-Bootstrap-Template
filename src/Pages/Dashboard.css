body {
    background: linear-gradient(to right, #1b82be, #0f5ea0);
    padding: 0;
    margin: 0;
    font-family: 'Lato', sans-serif;
    color: #000;
  }
  
  .master-container {
    display: flex;
    flex-direction: row;
    justify-content: center;
    margin: 0 auto;
    width: 200%;
    max-width: 1200px;
  }
  .dashboard-container {
    /* margin-top: -31.70%; */
    /* display: flex; */
    width: 100%;
    /* margin-top: 4%; */
    align-items: flex-start;
    padding: 20px;
    margin: 0 auto;
  }
  
  .dashboard-profile-section {
    width: 100%;
    max-width: 600px;
    background-color: #fff;
    border: 1px solid #ddd;
    border-radius: 8px;
    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
    padding: 20px;
    margin-top: 20px;
  }
  
  .edit-button {
    background-color: #0073b1;
    color: #fff;
    padding: 10px 20px;
    border-radius: 4px;
    cursor: pointer;
    font-size: 16px;
    margin-bottom: 20px;
    align-self: flex-end;
    margin-right: 0%;
    margin-left: 90%;
  }
  
  .edit-button:hover {
    background-color: #005a8e;
  }
  
  .skills-section {
    margin-bottom: 20px;
  }
  
  .skills-list {
    list-style: none;
    padding: 0;
  }
  
  .skills-list-item {
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding: 10px;
    border-bottom: 1px solid #ddd;
  }
  
  .add-section {
    display: flex;
    align-items: center;
    margin-top: 10px;
  }
  
  .add-section input[type="text"],
  .add-section textarea {
    flex: 1;
    padding: 8px;
    border: 1px solid #ddd;
    border-radius: 4px;
    margin-right: 10px;
  }
  
  .add-section button {
    background-color: #0073b1;
    color: #fff;
    border: none;
    padding: 8px 16px;
    border-radius: 4px;
    cursor: pointer;
  }
  
  .add-section button:hover {
    background-color: #005a8e;
  }
  
  .dashboard-Photo {
    /* display: flex; */
    width: 35%;
    border-radius: 100px;
    background-color: #ffffff;
    margin-top: 4%;
    height: 225px;
    
  }
  
  .dashboard-Photo .profile_img {
    width: 150px;
    height: 150px;
    object-fit: cover;
    margin: 10px auto;
    border: 8px solid #0664bb;
    border-radius: 50%;
  }
  .dashboard-Photo h3 {
    font-size: 16px;
    color: #5f2402;
    font-weight: 600;
    font-style: italic;
    font-variant: small-caps;
  }
  
  .dashboard-Photo p {
    font-size: 16px;
    color: #5f2402;
    font-weight: 600;
    font-style: italic;
    font-variant: small-caps;
  }
  
  