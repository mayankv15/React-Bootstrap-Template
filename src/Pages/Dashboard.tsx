import React from "react";
import "./Dashboard.css";
import Skills from "./Home/Skills";
import Education from "./Home/Education";
import About from "./Home/About";
import Header from "./Header";
import Experience from "./Home/Experience";
import { useNavigate } from "react-router-dom";
// import MyPic from "../MYPICTURE.jpg";
import { useSelector } from "react-redux";
import LoginPage from "./LoginPage";
import { Edit } from "@material-ui/icons";

const Dashboard: React.FC = () => {
  const Navigate = useNavigate();
  const login = useSelector((state: any) => state.user.login);

  const onEdit = () => {
    Navigate("/Profile");
  };

  if (!login) {
    Navigate("/");
  }

  return (
    <>
      {login ? (
        <div>
          <Header />
          <div className="master-container">
            <div className="dashboard-Photo">
              <div className="container">
                <div className="row">
                  <div className="col-lg-4">
                    <div className="card shadow-sm">
                      <div className="card-header bg-transparent text-center">
                        {/* <img
                          className="profile_img"
                          src={MyPic}
                          alt="profile"
                        /> */}
                        <h3>Mayank</h3>
                      </div>
                      <div className="card-body">
                        <p className="mb-0">
                          <strong className="pr-1">Current Company:</strong>____
                        </p>
                        <p className="mb-0">
                          <strong className="pr-1">Graduation Year:</strong>____
                        </p>
                        <p className="mb-0">
                          <strong className="pr-1">Place</strong>____
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>

            <div className="dashboard-container">
              <div className="dashboard-profile-section">
                {/* <button className="edit-button" onClick={onEdit}>
                  Edit
                </button> */}
                <Edit className="edit-button" onClick={onEdit} />
                <About edit={false} />
                <br />
                <Education edit={false} />
                <br />
                <Experience edit={false} />
                <br />
                <Skills edit={false} />
              </div>
            </div>
          </div>
        </div>
      ) : (
        (alert("Please Login First"),
        (
          <div>
            <LoginPage />
          </div>
        ))
      )}
    </>
  );
};

export default Dashboard;
