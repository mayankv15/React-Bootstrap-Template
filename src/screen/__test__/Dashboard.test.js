import React from "react";
import { render } from "@testing-library/react";
import Dashboard from "../Dashboard"

describe("Renders Dashboard", () => {
    it("should render Dashboard correctly", () => {
        const component = render(<Dashboard />);
        expect(component).toMatchSnapshot();
    });
});
