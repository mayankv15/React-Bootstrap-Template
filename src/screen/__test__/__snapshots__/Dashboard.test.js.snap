// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Dashboard should render Dashboard correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiGrid-root MuiGrid-container"
      >
        <div
          class="account-screen"
          id="account-screen"
          style="width: 100%;"
        >
          <div
            class="content-menu"
            id="content-menu"
          >
            <div
              class="main-menu"
              id="main-menu"
            >
              <ul>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="0"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="0"
                      >
                        Label
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="1"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="1"
                      >
                        Textarea
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="2"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="2"
                      >
                        Button
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="3"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="3"
                      >
                        Basic Tiles
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="4"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="4"
                      >
                        Radio
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="5"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="5"
                      >
                        Radio List
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="6"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="6"
                      >
                        Load Spinner
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="7"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="7"
                      >
                        Date Picker
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="8"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="8"
                      >
                        File Upload
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="9"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="9"
                      >
                        Toggle
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="10"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="10"
                      >
                        Popup Modal
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="11"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="11"
                      >
                        Select
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="12"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="12"
                      >
                        Select Box
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="13"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="13"
                      >
                        Check Box
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
                <li
                  class="menu-list-item"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text"
                    data-tab="14"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <span
                        class="menu-label"
                        data-tab="14"
                      >
                        Check Box List
                      </span>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
              </ul>
            </div>
          </div>
          <div
            class="content-body"
            id="content-body"
          >
            <div
              class="MuiGrid-root MuiGrid-container"
            >
              <div
                style="width: 100%;"
              >
                 
                <div
                  aria-labelledby="column-two-tabs-0"
                  id="column-two-tabpanel-0"
                  role="tabpanel"
                  style="display: flex; justify-content: center; margin-top: 10px; padding: 0px 5px;"
                >
                  <div
                    class="MuiBox-root MuiBox-root-2"
                  >
                    <p
                      class="MuiTypography-root makeStyles-label-3 undefined MuiTypography-body1"
                    >
                      Label
                       
                      *
                    </p>
                  </div>
                </div>
                <div
                  aria-labelledby="column-two-tabs-0"
                  id="column-two-tabpanel-0"
                  role="tabpanel"
                  style="display: flex; justify-content: center; margin-top: 10px; padding: 0px 5px;"
                >
                  <div
                    class="MuiBox-root MuiBox-root-4"
                  >
                    <p
                      class="MuiTypography-root makeStyles-othersLabelStyles-1 undefined MuiTypography-body1"
                    >
                      Custom styled Label
                       
                      
                    </p>
                  </div>
                </div>
                <div
                  aria-labelledby="column-two-tabs-0"
                  id="column-two-tabpanel-0"
                  role="tabpanel"
                  style="display: flex; justify-content: center; margin-top: 10px; padding: 0px 5px;"
                >
                  <div
                    class="MuiBox-root MuiBox-root-5"
                  >
                    <div>
                      <ol
                        style="font-weight: bold;"
                      >
                        <li>
                          First import the component by placing below line into your file.
                          <br />
                          <ul>
                            <em
                              style="font-weight: normal;"
                            >
                              import Label from "../components/common/Label"
                            </em>
                          </ul>
                        </li>
                        <br />
                        <li>
                          Then use the component like below:
                          <br />
                          <ul
                            style="font-weight: normal;"
                          >
                            <em>
                              &lt;Label title='Label' required /&gt;
                              <br />
                              &lt;Label inputLabelStyles={classes.othersLabelStyles} title='Custom styled Label'/&gt;
                            </em>
                          </ul>
                        </li>
                        <br />
                        <li>
                          Pass the props value -
                          <ul
                            style="font-weight: normal;"
                          >
                            <li>
                              <em>
                                title
                              </em>
                               : pass the title value into this variable.
                            </li>
                            <li>
                              optional props:
                              <ul>
                                <li>
                                  <em>
                                    inputLabelStyles
                                  </em>
                                   : custom style can be passed into this variable.
                                </li>
                                <li>
                                  <em>
                                    required/required=
                                    {true}
                                  </em>
                                   : can be added in the label component in order to make it mandatory. (E.g. Label*)
                                </li>
                              </ul>
                            </li>
                          </ul>
                        </li>
                      </ol>
                    </div>
                  </div>
                </div>
              </div>
               
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiGrid-root MuiGrid-container"
    >
      <div
        class="account-screen"
        id="account-screen"
        style="width: 100%;"
      >
        <div
          class="content-menu"
          id="content-menu"
        >
          <div
            class="main-menu"
            id="main-menu"
          >
            <ul>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="0"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="0"
                    >
                      Label
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="1"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="1"
                    >
                      Textarea
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="2"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="2"
                    >
                      Button
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="3"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="3"
                    >
                      Basic Tiles
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="4"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="4"
                    >
                      Radio
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="5"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="5"
                    >
                      Radio List
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="6"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="6"
                    >
                      Load Spinner
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="7"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="7"
                    >
                      Date Picker
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="8"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="8"
                    >
                      File Upload
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="9"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="9"
                    >
                      Toggle
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="10"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="10"
                    >
                      Popup Modal
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="11"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="11"
                    >
                      Select
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="12"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="12"
                    >
                      Select Box
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="13"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="13"
                    >
                      Check Box
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
              <li
                class="menu-list-item"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  data-tab="14"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="menu-label"
                      data-tab="14"
                    >
                      Check Box List
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </li>
            </ul>
          </div>
        </div>
        <div
          class="content-body"
          id="content-body"
        >
          <div
            class="MuiGrid-root MuiGrid-container"
          >
            <div
              style="width: 100%;"
            >
               
              <div
                aria-labelledby="column-two-tabs-0"
                id="column-two-tabpanel-0"
                role="tabpanel"
                style="display: flex; justify-content: center; margin-top: 10px; padding: 0px 5px;"
              >
                <div
                  class="MuiBox-root MuiBox-root-2"
                >
                  <p
                    class="MuiTypography-root makeStyles-label-3 undefined MuiTypography-body1"
                  >
                    Label
                     
                    *
                  </p>
                </div>
              </div>
              <div
                aria-labelledby="column-two-tabs-0"
                id="column-two-tabpanel-0"
                role="tabpanel"
                style="display: flex; justify-content: center; margin-top: 10px; padding: 0px 5px;"
              >
                <div
                  class="MuiBox-root MuiBox-root-4"
                >
                  <p
                    class="MuiTypography-root makeStyles-othersLabelStyles-1 undefined MuiTypography-body1"
                  >
                    Custom styled Label
                     
                    
                  </p>
                </div>
              </div>
              <div
                aria-labelledby="column-two-tabs-0"
                id="column-two-tabpanel-0"
                role="tabpanel"
                style="display: flex; justify-content: center; margin-top: 10px; padding: 0px 5px;"
              >
                <div
                  class="MuiBox-root MuiBox-root-5"
                >
                  <div>
                    <ol
                      style="font-weight: bold;"
                    >
                      <li>
                        First import the component by placing below line into your file.
                        <br />
                        <ul>
                          <em
                            style="font-weight: normal;"
                          >
                            import Label from "../components/common/Label"
                          </em>
                        </ul>
                      </li>
                      <br />
                      <li>
                        Then use the component like below:
                        <br />
                        <ul
                          style="font-weight: normal;"
                        >
                          <em>
                            &lt;Label title='Label' required /&gt;
                            <br />
                            &lt;Label inputLabelStyles={classes.othersLabelStyles} title='Custom styled Label'/&gt;
                          </em>
                        </ul>
                      </li>
                      <br />
                      <li>
                        Pass the props value -
                        <ul
                          style="font-weight: normal;"
                        >
                          <li>
                            <em>
                              title
                            </em>
                             : pass the title value into this variable.
                          </li>
                          <li>
                            optional props:
                            <ul>
                              <li>
                                <em>
                                  inputLabelStyles
                                </em>
                                 : custom style can be passed into this variable.
                              </li>
                              <li>
                                <em>
                                  required/required=
                                  {true}
                                </em>
                                 : can be added in the label component in order to make it mandatory. (E.g. Label*)
                              </li>
                            </ul>
                          </li>
                        </ul>
                      </li>
                    </ol>
                  </div>
                </div>
              </div>
            </div>
             
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
